torch/_higher_order_ops/cond.py

```diff
@@ -57,6 +58,7 @@ def __init__(self):
         super().__init__("cond")
 
     def __call__(self, pred, true_fn, false_fn, operands):
+        validate_lifted_arg_types(operands)
```

Comment:
The error message is kind of wonky here "lifted args must be of type ...", you probably want to pass in a string to change the "lifted args" part of it